v0.1/elements/electrodeArray.schema.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$id": "https://schema.hbp.eu/sand/0.1/elements/electrodeArray.schema.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "description": "SANDS definition of an electrode array",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "@type", 
+    "@id",
+    "identifier",
+    "electrodes"
+    ],
+  "properties": {
+    "@type": {
+      "type": "string",
+      "description": "Metadata node type (schema).",
+      "const": "https://schema.ebrains.eu/sands/ElectrodeArray"
+    },
+    "@id": {
+      "type": "string",
+      "description": "Metadata node identifier.",
+      "pattern": "sands/elements/electrodeArray/([A-Za-z0-9_-]+).json"
+    },
+    "identifier": {
+      "type": "string",
+      "description": "Identifier of this electrode array within files."
+    },
+    "electrodes":{
+	  "type": "array",
+	  "minItems": 1,
```

Comment:
Change "minItems" to 2 --> An electrode array consisting of 1 electrode is an `electrode`. The required items need to start at 2 to qualify for an `electrodeArray`. FYI wiki has the correct count ;)

Reply:
yes. should have bee two. I'll correct it